### PR TITLE
Include diagonal control

### PR DESCRIPTION
**Pull request by cua-cua**

### CODE REVIEW

File: control.c [hunk 1]
```diff
@@ -270,6 +281,7 @@ void motor_control(void)
 	float angular_pwm;
 	float side_sensors_feedback = 0.;
 	float front_sensors_feedback = 0.;
+	float diagonal_sensors_feedback;
```
Comment (Peque) [inline control.c]: `= 0.;`

File: solve.c [hunk 2]
```diff
@@ -107,7 +107,7 @@ void set_run_sequence(void)
 	run_sequence[i] = '\0';
 }
 
-/**
+/*
```
Comment (Peque) [inline solve.c]: ?

**Review by Peque — CHANGES_REQUESTED** [review]

File: control.c (hunk 1 above)
Comment (cua-cua) [reply control.c]: fccaa95f

File: solve.c (hunk 2 above)
Comment (cua-cua) [reply solve.c]: fccaa95f

**Review by Peque — APPROVED** [review]

**Pull request merged**